### PR TITLE
update package composer for Laravel 8

### PR DESCRIPTION
**Pull request by arborrow**
I have not tested but would not anticipate any breaking changes. Kindly consider pulling in these updates.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,9 +10,16 @@
         }
     ],
     "minimum-stability": "dev",
+    "repositories":
+    [
+            {
+                "type": "vcs",
+                "url": "https://github.com/arborrow/laravel-phone-validator"
+            }
+    ],
```
Comment (stuyam) [inline composer.json:19]: Im guessing this isn't supposed to be here?

Comment (arborrow) [reply composer.json:19]: Correct, I should have branched before I continued playing with my local branch

**Comment by stuyam** [comment]
@arborrow you have been opening issues and pull requests for a few years now. I think it's about time you get get to be a collaborator on this project. I spend more time with Ruby on Rails these days because of work so I appreciate the help from people still doing Laravel full time 👍

**Comment by arborrow** [comment]
@stuyam I'm happy to be of whatever help I can

**Comment by stuyam** [comment]
I just added you as a collaborator to this repo and the Laravel Kickbox Validator for email (https://github.com/stuyam/laravel-kickbox-validator), they are very similar. Don't feel an obligation to do anything, but something like bumping Laravel versions, if that is blocking for you and I am not around, feel free to do whatever you need 👍

**Comment by stuyam** [comment]
Releases are managed automatically with packagist via GitHub tags/releases if you even need to tag a new release.

**Pull request merged**